app/views/layouts/_site_header.html.erb

```diff
@@ -23,9 +23,13 @@
     <div class="wrap-local-nav">
       <div class="wrap-bar">
         <nav class="local-nav" aria-label="Main menu">
-          <% if user_signed_in? && current_user.admin? %>
-            <%= link_to("Process submissions", process_path, class: 'button-secondary') %>
-            <%= link_to("Admin", admin_root_path, class: 'button-secondary') %>
+          <% if user_signed_in? %>
+            <% if can? :process_theses, Thesis %>
```

Comment:
Ah... I was trying to write up this bug but wasn't sure what was going on enough to log it even. I had a local note to figure it out and open a bug. 💯